bindings/nodejs/lib/types/block/payload/signed_transaction/transaction.ts

```diff
@@ -96,36 +65,18 @@ class RegularTransactionEssence extends TransactionEssence {
         creationSlot: SlotIndex,
         contextInputs: ContextInput[],
         inputs: Input[],
-        inputsCommitment: HexEncodedString,
-        outputs: Output[],
         allotments: ManaAllotment[],
+        outputs: Output[],
         payload?: Payload,
```

Comment:
weird ordering

Reply:
I know... I wanted to change it but TS forces me to have the optionals at the end

Reply:
make it non-optional maybe? I mean you can still pass in null

Reply:
I can pass null if it's expecting a non optional Payload? Wtf?

Reply:
the ? mark means you can just not have that parameter at all in the call site. but you can pass null instead

Reply:
I think I'll keep it like this for now, this seems like a bigger discussion to have outside of this PR